acs-integration-tests/pom.xml

```diff
@@ -96,22 +99,29 @@
                     </properties>
                     <systemPropertyVariables>
                         <ACS_SERVICE_ID>predix-acs</ACS_SERVICE_ID>
```

Comment:
does changing the service Id to to keti or acs break execution of tests?

Reply:
Currently, acceptance profile is executed by the deploy pipeline and assumes predix and cloud foundry environment.  Not sure if we need acceptance tests for public version.  We could address it when the need arises.

Reply:
We could address service renaming and zone header (PREDIX_ZONE_ID) renaming at the time of renaming acs packages.  I will look though if we could use parameterize service name with environment variable.